tests/testthat/_snaps/method-register-dispatch.md

```diff
@@ -4,19 +4,19 @@
 
 # method lookup fails with informative messages
 
-    Can't find method for generic `foo()` with classes:
-    - x: <logical>
-    - y: <MISSING>
+    Can't find method for generic `foo()` with dispatch classes:
+    - x: logical, ANY
+    - y: MISSING
```

Comment:
I wonder if _always_ seeing ANY print is going to be annoying. Like, should we just assume it is implied?

Reply:
I think this is ok for now; it'll get fixed when we eventually print the original method here.